src/propOr.js

```diff
@@ -28,5 +29,5 @@ var _has = require('./internal/_has');
  *      favoriteWithDefault(alice);  //=> 'Ramda'
  */
 module.exports = _curry3(function propOr(val, p, obj) {
-  return _has(p, obj) ? obj[p] : val;
+  return (!isNil(obj) && _has(p, obj)) ? obj[p] : val;
```

Comment:
No, I do agree here.  `isNil` is useful to have on the public interface, but it really gains us nothing in readability, performance, or reuse when we use it internally.  I'll update soon.